pkg/harvester/models/harvesterhci.io.virtualmachineimage.js

```diff
@@ -105,6 +105,10 @@ export default class HciVmImage extends HarvesterResource {
       if (this.spec.sourceType === 'download') {
         return 'Downloading';
       } else if (this.spec.sourceType === 'upload') {
+        if (this.uploadError) {
+          return 'Failed';
+        }
+
```

Comment:
Nit: Is it better to remove else here for reading?

```js
if (imported?.status === 'Unknown') {
  if (this.spec.sourceType === 'download') {
    return 'Downloading';
  } 
  
  if (this.spec.sourceType === 'upload') {
    if (this.uploadError) {
      return 'Failed';
    }
    return 'Uploading';
  }
  
  return 'Exporting'
}
```

Reply:
Correct, thanks!